splunk-forwarder/README.md

```diff
@@ -0,0 +1,28 @@
+# Splunk forwarder for CloudWatch Log group subscriptions
+
+This module creates one or more CloudWatch subscriptions for one or more CloudWatch Log groups. 
+
+The module will provision a lambda function in the account that will act as a target for the subscriptions. The Lambda function artifact is located in the *telia-common-logs-prod* account and is maintained there. 
+
+The Lambda function forwards CloudWatch logs to a bucket called *telia-common-logs-prod-application-logs* in the common logs account. 
+
+The items in the bucket is later consumed using SQS based S3 ingestion by Splunk. 
```

Comment:
`The items in the bucket is later consumed using SQS based S3 ingestion by Splunk. ` - I would remove this from this description because this is not what this module is doing. I think it belongs to another README file somewhere else. :)